### PR TITLE
Add crictl to README

### PR DESCRIPTION
**Pull request by heartlock**
@xlgao-zju

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,15 +51,49 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ```bash
 go get github.com/kubernetes-incubator/cri-tools/cmd/critest
-$GOPATH/bin/critest
+go get github.com/kubernetes-incubator/cri-tools/cmd/critctl
 ```
-
-### Flags
+### critest
```
Comment (xlgao-zju) [inline README.md]: Also add introduction for critest.

**Comment by feiskyer** [comment]
@heartlock Could you group options and subcommands to a list? e.g. add `- ` for each item in markdown format.

**Comment by xlgao-zju** [comment]
> Could you group options and subcommands to a list? e.g. add - for each item in markdown format.

@heartlock Also add this for critest flags, plz.

**Comment by heartlock** [comment]
@xlgao-zju  updated, please have a look

File: README.md [hunk 2]
```diff
@@ -51,15 +51,61 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
...
 
-### Flags
+ validation test suites for kubelet CRI
 
-`--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
+#### Flags
 
-`--build-dependencies`, `-b`:If we don't need to build dependencies, we can add this flag `--build-dependencies=false` or `-b=false`.
+- `--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
```
Comment (feiskyer) [inline README.md]: missing image service address. Could you also change them to `--runtime-endpoint` and `--image-endpoint`? (consistent with crictl)

**Comment by xlgao-zju** [comment]
And do we need an example of `pod.json` and `container.json` ?

File: README.md (hunk 2 above)
Comment (heartlock) [inline README.md]: sure, do this in another pr

**Comment by feiskyer** [comment]
> And do we need an example of pod.json and container.json ?

yes, please

File: README.md [hunk 3]
```diff
@@ -51,15 +51,96 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ```bash
 go get github.com/kubernetes-incubator/cri-tools/cmd/critest
-$GOPATH/bin/critest
+go get github.com/kubernetes-incubator/cri-tools/cmd/critctl
 ```
+### critest
 
-### Flags
+ validation test suites for kubelet CRI
```
Comment (feiskyer) [inline README.md]: Cap the first letter.

File: README.md [hunk 4]
```diff
@@ -51,15 +51,96 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
...
 
-`--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
+#### Flags
 
-`--build-dependencies`, `-b`:If we don't need to build dependencies, we can add this flag `--build-dependencies=false` or `-b=false`.
+- `--runtime-endpoint`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-endpoint=/var/run/frakti.sock`. And we can run this test against frakti.
```
Comment (feiskyer) [inline README.md]: `--runtime-endpoint`, `-r`: CRI runtime server endpoint. Default is `/var/run/dockershim.sock`.

File: README.md [hunk 5]
```diff
@@ -51,15 +51,96 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
...
+- Run podsandbox with config file
+
+`crictl sandbox run pod.json`
+
+example of config file for podsandbox `pod.json`:
+
+```
+{
+    "metadata": {
+        "name": "podsandbox-1",
```
Comment (feiskyer) [inline README.md]: please add all fields of PodSandboxMetadata

File: README.md [hunk 6]
```diff
@@ -51,15 +51,96 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
...
+
+- `image`:Manage image
+
+- `exec`:Exec(exec, syncexec) a command in a running container
+
+- `portforward`:Forword ports(localport:remoteport) from a sandbox
+
+- `help`, `h`:Shows a list of commands or help for one command
+
+#### flags
```
Comment (feiskyer) [inline README.md]: rename to `global flags`

those are global flags, which must be specified before any subcommands.

**Comment by heartlock** [comment]
@feiskyer updated

File: README.md [hunk 7]
```diff
@@ -51,15 +51,99 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
...
+
+- `--ginkgo-flags`,`-g`:Space-separated list of arguments to pass to Ginkgo test runner.
+
+- `--focus`,`-f`:CRI e2e test will only run the test that match the focus regular expression.
+
+### crictl
+
+CLI for kubelet CRI
+
+- Run podsandbox with config file
```
Comment (feiskyer) [inline README.md]: Please move those two examples after Global flags and a header `#### Examples` for them.

**Comment by heartlock** [comment]
@feiskyer updated

**Comment by feiskyer** [comment]
Thanks. LGTM.
